### PR TITLE
plan/explain: add explain info for topn.

### PR DESCRIPTION
**Pull request by winoros**
We missed the explain info of topn.
PTAL @zz-jason @lamxTyler @hanfei1991

### CODE REVIEW

**Comment by zz-jason** [comment]
@winoros better to add some test to https://github.com/pingcap/tidb-test

**Comment by coocood** [comment]
LGTM

**Comment by winoros** [comment]
@zz-jason  already a pending pr.

File: plan/explain.go [hunk 1]
```diff
@@ -304,3 +304,20 @@ func (p *PhysicalMergeJoin) ExplainInfo() string {
 	}
 	return buffer.String()
 }
+
+// ExplainInfo implements PhysicalPlan interface.
+func (p *TopN) ExplainInfo() string {
+	buffer := bytes.NewBufferString("")
+	for i, item := range p.ByItems {
+		order := "asc"
```
Comment (hanfei1991) [inline plan/explain.go]: We can define explain info for by item. There exists same logic in `sort` and `topn`

**Pull request merged**